debug/render-template.js

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+"use strict";
+
+var config, container, data, options, path, template, util;
+
+/**
+ * Resolves a path and loads a file with require()
+ *
+ * @param {string} filePath
+ * @return {*} whatever is returned from require()
+ */
+function loadFile(filePath) {
+    var resultPath;
+
+    resultPath = path.resolve(process.cwd(), filePath);
+    console.error(`Loading file: ${resultPath}`);
```

Comment:
Because I want it to go to stderr so I can pipe the output of the program to a file if needed.